packages/client/src/components/measurements/MeasurementsTable.re

```diff
@@ -120,13 +120,87 @@ module Helpers = {
       |> E.O.React.defaultNull;
   };
 
+  module MomentDate = {
+    [@react.component]
+    let make = (~date) =>
+      date
+      |> E.O.fmap(d =>
+           <div className=Styles.date>
+             {d |> MomentRe.Moment.fromNow(~withoutSuffix=None) |> Utils.ste}
+           </div>
+         )
+      |> E.O.default(<Null />);
+  };
+
+  module RelevantAt = {
+    [@react.component]
+    let make = (~m: Types.measurement) =>
+      m.relevantAt
+      |> E.O.fmap(d =>
+           <div className=Styles.date>
+             {d |> MomentRe.Moment.fromNow(~withoutSuffix=None) |> Utils.ste}
+           </div>
+         )
+      |> E.O.default(<Null />);
+  };
+
+  module MeaurementVotes = {
+    module Styles = {
+      open Css;
+      let rightBlock = style([textAlign(`right)]);
+      let row =
+        style([
+          height(`em(3.)),
+          justifyContent(`spaceBetween),
+          alignItems(`center),
+        ]);
+      let subRow = style([width(`percent(100.))]);
+    };
+    module FC = ForetoldComponents;
+    [@react.component]
+    let make = (~measurement: Types.measurement) => {
+      MeasurementVotesGet.component(
+        ~measurementId=?Some(measurement.id), measurementVotes =>
+        measurementVotes
+        |> Array.mapi((index, measurementVote: Types.vote) => {
```

Comment:
I think I'd prefer we use E.Array instead of Array. I don't trust the standard library that much.

Reply:
Me too. Fixed.